ui/src/app/views/sign-in-flow-template/component.tsx

```diff
@@ -0,0 +1,129 @@
+import {Component, OnChanges, OnInit} from '@angular/core';
+import {Router} from '@angular/router';
+
+import {SignInService} from 'app/services/sign-in.service';
+import {withWindowSize} from 'app/utils';
+import {InvitationKeyReact} from 'app/views/invitation-key/component';
+import {LoginReactComponent} from 'app/views/login/component';
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import {Content, Header, SignedIn, Template} from './style';
+
+interface ImagesSource {
+  backgroundImgSrc: string;
+  smallerBackgroundImgSrc: string;
+}
+
+interface PageTemplateState {
+  currentStep: number;
+  invitationKey: string;
+}
+
+@withWindowSize()
+export class SignPageTemplateReact extends React.Component<any, PageTemplateState> {
+  state: PageTemplateState;
```

Comment:
BTW, since you already declared the `PageTemplateState` in the angle brackets above, there's no need to also declare it here.